cmd/ocm/tunnel/cmd.go

```diff
@@ -65,14 +77,21 @@ func run(cmd *cobra.Command, argv []string) error {
 			clusterKey,
 		)
 	}
-
 	path, err := exec.LookPath("sshuttle")
 	if err != nil {
 		return fmt.Errorf("to run this, you need install the sshuttle tool first")
 	}
 
 	// Create the client for the OCM API:
-	connection, err := ocm.NewConnection().Build()
+	config, err := config.Load()
+	if err != nil {
```

Comment:
Also need to check if config == nil, as config.Load() might return an empty object.
```
 ./ocm tunnel hive-stage-01 --url https://api.openshift.com 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x88 pc=0x9d7c3a]

goroutine 1 [running]:
github.com/openshift-online/ocm-cli/cmd/ocm/tunnel.run(0x1154120, 0xc0001eb8f0, 0x1, 0x3, 0x0, 0x0)
	/home/siwu/git/haoran/ocm-cli/cmd/ocm/tunnel/cmd.go:91 +0x19a
github.com/spf13/cobra.(*Command).execute(0x1154120, 0xc0001eb8c0, 0x3, 0x3, 0x1154120, 0xc0001eb8c0)
	/home/siwu/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0x1154900, 0xc000108060, 0x4, 0x4)
	/home/siwu/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	/home/siwu/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
	/home/siwu/git/haoran/ocm-cli/cmd/ocm/main.go:129 +0x1ec

$ ls ~/.ocm.json 
ls: cannot access '/home/siwu/.ocm.json': No such file or directory
```

Reply:
done

Reply:
works!